src/main/java/uk/gov/ida/stub/idp/domain/EidasUser.java

```diff
@@ -54,4 +57,22 @@ public void setAddress(EidasAddress address) {
     public void setGender(Gender gender) {
         this.gender = gender;
     }
+
+    @Override
+    public boolean equals(Object o) {
```

Comment:
I added this equals specifically to ensure that the unit test passes (as during deserialization Jackson creates a new but empty `EidasUser`).